shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -68,6 +68,58 @@ - (instancetype)initWithConnection:(NSNumber*)connection
 
 #pragma mark -
 
+// The information of a view that has been attached to the engine.
+@interface FlutterEngineViewRecord : NSObject
+- (instancetype)initWithController:(nonnull FlutterViewController*)controller
+                            engine:(nonnull FlutterEngine*)engine
+                  semanticsEnabled:(BOOL)semanticsEnabled;
+
+- (BOOL)semanticsEnabled;
+
+- (void)setSemanticsEnabled:(BOOL)semanticsEnabled;
+
+@property(nonatomic, readonly, weak) FlutterViewController* controller;
+
+@property(nonatomic, readonly) std::shared_ptr<flutter::AccessibilityBridgeMac> bridge;
+@end
+
+@implementation FlutterEngineViewRecord {
+  __weak FlutterEngine* _engine;
+}
+
+- (instancetype)initWithController:(FlutterViewController*)controller
+                            engine:(nonnull FlutterEngine*)engine
+                  semanticsEnabled:(BOOL)semanticsEnabled {
+  self = [super init];
+  if (self != nil) {
+    _controller = controller;
+    _engine = engine;
+    [self setSemanticsEnabled:semanticsEnabled];
+  }
+  return self;
+}
+
+- (BOOL)semanticsEnabled {
+  return !!_bridge;
+}
+
+- (void)setSemanticsEnabled:(BOOL)semanticsEnabled {
+  if (semanticsEnabled == [self semanticsEnabled]) {
+    return;
+  }
+  if (semanticsEnabled) {
+    _bridge = [_engine createAccessibilityBridge:_engine viewController:_controller];
```

Comment:
If the accessibility bridge is tied to the view controller, does it make sense for the engine to create accessibility bridges? Should we move `createAccessibilityBridge:` to `FlutterEngineViewRecord` or `FlutterViewController`?

Reply:
The bridge has been moved to `FVC`, as well as `createAccessibilityBridge`.